GPU/GPUCommonHW.h

```diff
@@ -62,6 +62,7 @@ class GPUCommonHW : public GPUCommon {
 
 	void Execute_TexFlush(u32 op, u32 diff);
 
+	// TODO: Have these return an error code if they jump to a bad address. If bad, stop the FastRunLoop.
```

Comment:
Or just set `gpuState = GPU_ERROR;` and set `downcount = 0;` to force a recheck.

-[Unknown]

Reply:
Oh yeah, that's probably the slickest solution.

Reply:
Actually it seems we already do that in many cases. But not in fast memory mode.

I am going to remove Execute_JumpFast and Execute_CallFast. I don't believe they are worth it.